Include/internal/pycore_object.h

```diff
@@ -24,6 +24,37 @@ extern "C" {
         .ob_size = size, \
     }
 
+static inline void
+_Py_DECREF_SPECIALIZED(PyObject *op, const destructor destruct)
+{
+#ifdef Py_REF_DEBUG
+    _Py_RefTotal--;
+#endif
+    if (--op->ob_refcnt != 0) {
+        assert(op->ob_refcnt > 0);
+    }
+    else {
+#ifdef Py_TRACE_REFS
+        _Py_ForgetReference(op);
+#endif
+        destruct(op);
+    }
+}
+
+static inline void
+_Py_DECREF_IMMORTAL(PyObject *op)
+{
+#ifdef Py_REF_DEBUG
+    _Py_RefTotal--;
+#endif
+    op->ob_refcnt--;
+#ifdef Py_DEBUG
+    if (op->ob_refcnt <= 0) {
+        // Calls _Py_FatalRefcountError for None, True, and False
+        _Py_Dealloc(op);
```

Comment:
Can you change this to call `_Py_FatalRefcountError` directly? 
We might want to use `_Py_DECREF_IMMORTAL` for other "immortal" objects like `0` or `()` and there is no guarantee that their de-allocation functions will call `_Py_FatalRefcountError`.

Reply:
Done. I moved `_Py_FatalRefcountError` from `pycore_pyerrors` to `pycore_object` as well, rather than having an `#include`-dependence between the pycore_files.